js/src/api/contract/contract.js

```diff
@@ -232,51 +233,102 @@ export default class Contract {
       return this._subscribe(event, options, callback);
     };
 
+    event.register = (options = {}, callback) => {
```

Comment:
I agree with this. First I wanted to call it `manualSubscribe` or something like that.